scripts/NuclearCraft.zs

```diff
@@ -1,4 +1,5 @@
 import mods.jei.JEI.removeAndHide as rh;
+	print("--- loading NuclearCraft.zs ---");
```

Comment:
So only formatting changed in this script?

Reply:
Yep